controllers/etcd_controller_test.go

```diff
@@ -229,6 +229,8 @@ var _ = Describe("Druid", func() {
 			setStatefulSetReady(&sts)
 			err = c.Status().Update(context.TODO(), &sts)
 			Expect(err).NotTo(HaveOccurred())
+			// sleep needed here as we need to wait till etcd resource is added to the queue
+			time.Sleep(time.Second * 2)
```

Comment:
looks like you will have to increase the time here.

Reply:
Changed the logic to get rid of the sleep. It was dirty anyway :D